package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "carbon-react",
-  "version": "4.5.1",
```

Comment:
The package-lock.json should also update when this is changed.

Reply:
@chrisbarber86 Updated now